content/blog/2023/10/kubewarden-1.8-release.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Kubewarden 1.8.0 release
+authors:
+  - Víctor Cuadrado
+date: 2023-10-11
+---
+
+Today we are happy to announce the release of Kubewarden 1.8.0! 🎉🥳
+
+This is a small release, focused on [OpenTelemetry](https://opentelemetry.io/).
+
+The OpenTelemetry Protocol (OTLP) got its [first 1.0.0
+version](https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.0.0)
+in July 2023, and the OpenTelemetry stack is becoming more stable; several
```

Comment:
```suggestion
in July 2023; several
```

It seems a bit contradictory/confusing to read on one line that otel is becoming more stable, but then on the next paragraph to read the entire opposite. So either we use a synonym, or we just drop that like in the suggestion from above.